src/main/java/org/atlasapi/query/content/search/SherlockSearchResolver.java

```
@@ -0,0 +1,172 @@
+import org.atlasapi.media.entity.Specialization;
+import org.atlasapi.persistence.content.ContentResolver;
+import org.atlasapi.persistence.content.ResolvedContent;
+import org.atlasapi.persistence.content.SearchResolver;
+import org.atlasapi.persistence.lookup.entry.LookupEntry;
+import org.atlasapi.persistence.lookup.entry.LookupEntryStore;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class SherlockSearchResolver implements SearchResolver {
+
+    private static final ContentMapping CONTENT = IndexMapping.getContent();
+
+    private final ContentSearcher searcher;
+    private ContentResolver contentResolver;
+    private LookupEntryStore lookupEntryStore;
+    private final long timeout;
+    private final NumberToShortStringCodec idCodec;
+
+    public SherlockSearchResolver(
+            ContentSearcher searcher,
+            long timeout,
+            ContentResolver contentResolver,
+            LookupEntryStore lookupEntryStore,
+            NumberToShortStringCodec idCodec
+    ) {
+        this.searcher = checkNotNull(searcher);
+        this.timeout = timeout;
+        this.contentResolver = contentResolver;
+        this.lookupEntryStore = lookupEntryStore;
+        this.idCodec = idCodec;
+    }
+
+    @Override
+    public List<Identified> search(
+            org.atlasapi.search.model.SearchQuery owlQuery,
+            Application application
+    ) {
+        SearchParameter titleSearchParameter = SearchParameter.builder()
+                .withMapping(CONTENT.getTitle())
+                .withValue(owlQuery.getTerm())
+                //.withBoost(owlQuery.getTitleWeighting())
+                .build();
+
+        // TODO Specific weightings are not yet supported in Sherlock
```

Should we throw exceptions if any of the unsupported fields so that it becomes clear if someone tries to use them that they can't, instead of us silently ignoring them?

---

We decided not to do this since we are not currently using boostings in equiv, adding these will be part of the migration of search